### DISCUSSION TITLE
Gradient clipping before unscaling

### DISCUSSION THREAD

**Discussion by marcovisentin** [milesial/Pytorch-UNet#467] [Q&A]
At lines 114-115 in train.py gradients are clipped before unscaling. I think it's a mistake but I am not very familiar with AMP so I might be missing something.
